package-lock.json

```diff
@@ -4384,12 +4384,14 @@
         "balanced-match": {
```

Comment:
Since this PR doesn't update package.json, we do not need to update package-lock.json. Can you please remove the changes to this file?